app/main.ipynb

```diff
@@ -30,38 +30,28 @@
    }
   },
   {
+   "metadata": {},
    "cell_type": "code",
-   "execution_count": null,
    "outputs": [],
+   "execution_count": null,
    "source": [
-    "# write your code for task 0 here"
-   ],
-   "metadata": {
-    "collapsed": false,
-    "pycharm": {
-     "name": "#%%\n"
-    }
-   }
-  },
-  {
-   "cell_type": "markdown",
-   "source": [
-    "# Task 1\n",
-    "Print the count of men and women in the dataset."
-   ],
-   "metadata": {
-    "collapsed": false,
-    "pycharm": {
-     "name": "#%% md\n"
-    }
-   }
+    "df = pd.read_csv(\"../data/adult.csv\")\n",
+    "df = df.replace(\"?\", np.nan)\n",
+    "df = df.dropna()\n",
+    "df = df[df[\"salary\"].isin([\"<=50K\", \">50K\"])]\n",
+    "df[\"salary K$\"] = pd.to_numeric(df[\"salary K$\"], errors=\"coerce\")\n",
+    "df = df.dropna(subset=[\"salary K$\"])\n",
+    "df"
+   ]
```

Comment:
you should check data for correctness. So if salary = "<=50K", then you should ensure that column "salary K$" has an appropriate value